src/index.ts

```diff
@@ -1,22 +1,30 @@
 /* eslint-disable no-console */
 import express, { Application, Request, Response } from 'express';
 import mongoose from 'mongoose';
+/* import User, { IUser } from './models/user'; */
+import accountsRouter from './accounts/account.routes';
 
 // Boot express
 const app: Application = express();
 const port = 5000;
 
-mongoose.connect('mongodb://localhost/tropogs', { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.connect('mongodb://localhost/tropogs', {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
 const db = mongoose.connection;
 db.on('error', console.error.bind(console, 'connection error:'));
 db.once('open', () => {
   console.log('connected to db!');
 });
 
 // Application routing
-app.use('/', (req: Request, res: Response) => {
-  res.status(200).send({ data: 'Hello from the other side' });
+app.use('/', (req: Request, res: Response, next) => {
+  // res.status(200).send({ data: 'Hello from the other side' });
```

Comment:
Do not comment out. Maybe just change the route to `/ping` .

Reply:
Fixed.